test_scripts/upgrade.sh

```diff
@@ -20,104 +27,136 @@ echo "Checking out $START_BRANCH branch"
 git checkout $START_BRANCH
 
 ## Change to molecule directory on pre 7.0 branches
-
-cd roles/confluent.test/
+if [[ $START_BRANCH != 7* ]]
+then
+  cd roles/confluent.test/
+fi
 
 ## Run Molecule Converge on scenario
 echo "Running molecule converge on $SCENARIO_NAME"
 molecule converge -s $SCENARIO_NAME
 
 ## Change to base of cp-ansible
-cd ../..
+if [[ $START_BRANCH != 7* ]]
+then
+  cd ../..
+fi
 
 ## Checkout ending branch
 echo "Checkout $END_BRANCH branch"
 git checkout $END_BRANCH
 
+## With 7.0.x onwards, cp-ansible is packaged as an ansible collection and hence we need to specify collection path to enable running playbooks
+## https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths
+export ANSIBLE_COLLECTIONS_PATH=../../
```

Comment:
Is this true for all versions ?

Reply:
No, it isn't. Before 7.x.y, it won't be used. I'll wrap this under a conditional too